src/Compute/Compute/Extension/SetAzureVMExtensionBaseCmdlet.cs

```diff
@@ -69,15 +67,10 @@ public class SetAzureVMExtensionBaseCmdlet : VirtualMachineExtensionBaseCmdlet
             HelpMessage = "Disable auto-upgrade of minor version")]
         public SwitchParameter DisableAutoUpgradeMinorVersion { get; set; }
 
-        [Parameter(
-            Mandatory = false)]
-        public bool? EnableAutomaticUpgrade { get; set; }
```

Comment:
Remove a optional parameter is also a breaking change. Is it necessary?

Reply:
its not removed. I moved from this file (basecmdlet) to the Set-AzVmExtension file. 
Because other 3 cmdlets inheriting from the basecmdlet did not need this parameter. It was wrong design.

Reply:
Yes, I have resolved this.